receiver/splunkhecreceiver/receiver_test.go

```diff
@@ -196,18 +196,6 @@ func Test_splunkhecReceiver_handleReq(t *testing.T) {
 				assert.Equal(t, responseNotFound, body)
 			},
 		},
-		{
-			name: "incorrect_content_type",
```

Comment:
Do you think it's worth keeping this test, with the assertion checking success?

Reply:
+1. Since it is now valid to have the wrong content type let's have a test that verifies it is indeed a valid case.

Reply:
Added, all tests pass :) Thanks for the review!